src/kirby/components/loading-overlay/loading-overlay.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'kirby-loading-overlay',
+  templateUrl: './loading-overlay.component.html',
+  styleUrls: ['./loading-overlay.component.scss'],
+})
+export class LoadingOverlayComponent {
+  @Input() public isLoading = true;
```

Comment:
I think all our `boolean` inputs are using a naming scheme without `is..`, `has..` and just writes the name plain. 
So in this case it would be `loading`. I think we should align that.

Reply:
Seems like we do both, eg. sizing directive
```
  @HostBinding('class.sm')
  isSmallSize: boolean;
  @HostBinding('class.md')
  isMediumSize: boolean;
  @HostBinding('class.lg')
  isLargeSize: boolean;
  @Input() set size(size: Sizes) {
    if (size === 'sm') {
      this.isSmallSize = true;
    }
    if (size === 'md') {
      this.isMediumSize = true;
    }
    if (size === 'lg') {
      this.isLargeSize = true;
    }
  }
```

I would prefer is prefix for boolean because it is more explainable, what the variable contains.